cns/cnsclient/cli.go

```diff
@@ -84,7 +88,7 @@ func getCmd(client *CNSClient, arg string) error {
 }
 
 // Sort the addresses based on IP, then write to stdout
-func printIPAddresses(addrSlice []cns.IPAddressState) {
+func printIPAddresses(addrSlice []cns.IPConfigurationStatus) {
```

Comment:
Now that the other CNS PR has merged, we can use this to pretty print

https://github.com/Azure/azure-container-networking/pull/806/files#diff-f17e0d823dd4a062ed017125571f6268fa5a204979b4140cf0f73363576a1457R60

Reply:
added it in 2nd commit